eras/conway/impl/test/Test/Cardano/Ledger/Conway/BinarySpec.hs

```diff
@@ -28,4 +30,9 @@ spec = do
     roundTripCborSpec @(Voter StandardCrypto)
     roundTripConwayCommonSpec @Conway
     -- ConwayGenesis only makes sense in Conway era
-    roundTripEraSpec @Conway @(ConwayGenesis StandardCrypto)
+    context "Genesis" $ do
+      roundTripCborSpec @(UpgradeConwayPParams Identity)
+      roundTripCborSpec @(Constitution (ConwayEra StandardCrypto))
+      roundTripCborSpec @(Committee (ConwayEra StandardCrypto))
+      roundTripCborSpec @(ConwayGenesis StandardCrypto)
```

Comment:
This test is totally superfluous, since it will test roundtrip for all protocol versions, which is unnecessary. See the comment above:
> ConwayGenesis only makes sense in Conway era

Therefore we only test `ConwayGenesis` for Conway era, which is done below

Reply:
:memo: got it! Thanks! :+1: